charaData.json

```diff
@@ -659,6 +659,50 @@
         "imageURL": "./charaimgs/3040169000_01.png",
         "en": "Nier"
     },
+    "モニカ(リミテッドver)": {
+        "name": "モニカ(リミテッドver)",
+        "ja": "モニカ(リミテッドver)",
+        "element": "wind",
+        "type": "balance",
+        "race": "human",
+        "sex": "female",
+        "fav1": "sword",
+        "fav2": "none",
+        "support": "none",
+        "support2": "none",
+        "support3": "none",
+        "minhp": "300",
+        "hp": "1400",
+        "minattack": "1500",
+        "attack": "9000",
+        "baseDA": 7.0,
+        "baseTA": 3.0,
+        "ougiRatio": 4.5,
+        "imageURL": "./charaimgs/3040238000_01.png",
+        "en": "Monika (Grand)"
+    },
+    "ジークフリート(火属性ver)": {
+        "name": "ジークフリート(火属性ver)",
+        "ja": "ジークフリート(火属性ver)",
+        "element": "fire",
+        "type": "pecu",
+        "race": "human",
+        "sex": "male",
+        "fav1": "sword",
+        "fav2": "none",
```

Comment:
Siegfried's fav2 is also katana.
https://livedoor.blogimg.jp/hecate_mk2-gbf/imgs/e/5/e5b82a62.jpg